pslogin/src/main/scala/WorldSessionActor.scala

```diff
@@ -4117,6 +4389,76 @@ class WorldSessionActor extends Actor with MDCContextAware {
                         }
                       }
                     }
+                    else if(kit.Definition == GlobalDefinitions.super_medkit) {
+                      if(player.Health == player.MaxHealth) {
+                        sendResponse(ChatMsg(ChatMessageType.UNK_225, false, "", "@HealComplete", None))
+                      }
+                      else if(System.currentTimeMillis - whenUsedLastSMKit < 1200000) {
```

Comment:
Yikes. All of the magic comparisons to millis will need to go on future patches. We need a nice timer class

Reply:
By "go" meaning we should fix them up later, not as in on this PR

Reply:
Agreed